source/faq.txt

```diff
@@ -12,6 +12,27 @@ Frequently Asked Questions
    :depth: 2
    :class: singlecol
 
+How to fix MongoServerSelectionError: connect ECONNREFUSED ::1:27017 error?
+---------------------------------------------------------------------------
+
+Beginning with version 17, Node defaults to using ``IPV6``. You have a few
```

Comment:
Issue: The term "Node" doesn't seem to be a common way to refer to Node.js (e.g. always referred to as Node.js on https://nodejs.dev.

Suggestion: replace "Node" with "Node.js" or "Node.js driver" (if it's the driver and not the language).